### ISSUE TITLE
'csc_matrix' object has no attribute 'A'

### ISSUE THREAD

**Issue by mbassalbioinformatics** [stefanpeidli/mousipy#7]
Hi there, when trying to use translate, it fails with the following messages and error.

Any suggestions what might be the issue?

```
humanized_cebpa = translate(dd_scRNA)
100%|██████████| 25803/25803 [00:28<00:00, 898.55it/s]
Found direct orthologs for 15766 genes.
Found multiple orthologs for 465 genes.
Found no orthologs for 8410 genes.
Found no index in biomart for 1162 genes.
Traceback (most recent call last):

  Cell In[12], line 1
    humanized_data = translate(dd_scRNA)

  File ~/miniforge3/envs/scRNA/lib/python3.11/site-packages/mousipy/mousipy.py:326 in translate
    bdata = translate_multiple(bdata, adata, multiple, stay_sparse=stay_sparse, verbose=verbose)

  File ~/miniforge3/envs/scRNA/lib/python3.11/site-packages/mousipy/mousipy.py:215 in translate_multiple
    X = make_dense(adata.X)

  File ~/miniforge3/envs/scRNA/lib/python3.11/site-packages/mousipy/mousipy.py:23 in make_dense
    return X.A

AttributeError: 'csc_matrix' object has no attribute 'A'
```

even if i try to run the example provided
```
translate( scv.datasets.pancreas() )
```
still get the same error

Much appreciated!

**Comment by pakiessling** [comment]
Hi this is a problem with an update to the scipy version.
Until my pull request is merged you can install scipy < 1.14
or change the single line of code from my fork 😅

**Comment by stefanpeidli** [comment]
@mbassalbioinformatics thanks for raising this. @pakiessling is right and the suggested PR will fix this. I will merge it to main soon, thanks for the contribution  ❤️
